src/applications/disability-benefits/526EZ/pages/privateMedicalRecordRelease.js

```diff
@@ -0,0 +1,173 @@
+
+export const uiSchema = {
+  disabilities: {
+    items: {
+      'ui:description': recordReleaseDescription,
+      'ui:title': disabilityNameTitle,
+      'view:limitedConsent': {
+        'ui:title': limitedConsentTitle,
+      },
+      limitedConsent: {
+        'ui:title': limitedConsentTextTitle,
+        'ui:options': {
+          expandUnder: 'view:limitedConsent',
+          expandUnderCondition: true,
+        },
+        'ui:required': (formData, index) =>
+          _.get(`disabilities.${index}.view:limitedConsent`, formData),
+      },
+      'view:privateRecordsChoiceHelp': {
+        'ui:description': limitedConsentDescription,
+      },
+      providerFacility: {
+        'ui:options': {
+          itemName: 'Provider',
+          viewField: PrivateProviderTreatmentView,
+          hideTitle: true,
+        },
+        items: {
+          providerFacilityName: {
+            'ui:title': 'Name of private provider or hospital',
+            'ui:errorMessages': {
+              pattern: 'Provider name must be less than 100 characters.',
+            },
+          },
+          'ui:validations': [validateDate],
+          treatmentDateRange: dateRangeUI(
+            'Approximate date of first treatment',
+            'Approximate date of last treatment',
+            'End of treatment must be after start of treatment',
+          ),
+          providerFacilityAddress: Object.assign(addressUI('', false), {
+            street2: {
+              'ui:title': 'Street 2',
+            },
+            postalCode: {
+              'ui:title': 'Postal Code',
+              'ui:options': {
+                widgetClassNames: 'usa-input-medium',
+              },
+              'ui:validations': [
+                {
+                  validator: validateZIP,
+                },
+              ],
+            },
+          }),
+        },
+      },
+    },
+  },
+};
+
+export const schema = {
+  type: 'object',
+  properties: {
+    disabilities: {
+      type: 'array',
+      items: {
+        type: 'object',
+        required: ['providerFacility'],
+        properties: {
+          providerFacility: {
+            type: 'array',
+            minItems: 1,
+            maxItems: 100,
+            items: {
+              type: 'object',
+              required: ['providerFacilityName', 'providerFacilityAddress'],
+              properties: {
+                providerFacilityName: {
+                  type: 'string',
+                  minLength: 1,
+                  maxLength: 100,
+                },
+                treatmentDateRange: {
+                  type: 'object',
+                  properties: {
+                    from: {
+                      $ref: '#/definitions/date',
+                    },
+                    to: {
+                      $ref: '#/definitions/date',
+                    },
+                  },
+                  required: ['from', 'to'],
+                },
+                providerFacilityAddress: {
```

Comment:
Would it make sense to use the [common address `schema` and `uiSchema`](https://github.com/department-of-veterans-affairs/vets-website/blob/6a27450abed0eb724d9fa0334633ff8ffe700f78/src/platform/forms/definitions/address.js) for this?

Reply:
I tried to use that, but I think I kept hitting a wall with a schema error. It kept hiccuping on Street 2 and Steet being extraneous I believe. Then, either the postal code or the privacy agreement being in the schema was causing another issue.